### PR TITLE
Add uncategorised group (Not grouped) on the app navigation #874

### PR DESCRIPTION
**Pull request by zero-24**
Please see: #874 

Hi @rakekniven and @skjnldsv this here should solve the issue described and add back the `Not grouped` entry.

Can you please tell me whether this is the best approach and whether you would more like two `if's` so the same code without the `else` case.

And given that `Not grouped` does not seam to be in the translation files can you please tell me the best approach to add that back?

### CODE REVIEW

File: src/store/groups.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const mutations = {
 	extractGroupsFromContacts(state, contacts) {
 		// iterate contacts
 		contacts.forEach(contact => {
-			if (contact.groups) {
+			if (contact.groups.length != 0) {
```
Comment (skjnldsv) [inline src/store/groups.js]: please use `!==` :)

File: src/store/groups.js [hunk 2]
```diff
@@ -49,6 +49,20 @@ const mutations = {
 					group.contacts.push(contact.key)
 				})
 			}
+			else
```
Comment (skjnldsv) [inline src/store/groups.js]: please use inline else-if code styling

File: src/store/groups.js (hunk 1 above)
Comment (skjnldsv) [inline src/store/groups.js]: please use `!==` :)

File: src/store/groups.js [hunk 3]
```diff
@@ -49,6 +49,20 @@ const mutations = {
...
+			{
+				let notGroupedName = t('contacts', 'Not grouped');
```
Comment (skjnldsv) [inline src/store/groups.js]: you can use `const` since this is an immutable variable :)

**Comment by zero-24** [comment]
Thanks for your review @skjnldsv the requested changes has just been made.

**Comment by skjnldsv** [comment]
@zero-24 I just noticed the 'Not Groupped' appear in the `add to group` list. Could you fix it? :)

**Comment by skjnldsv** [comment]
@zero-24 please see #914

**Pull request merged**